### ISSUE TITLE
File looks fine on firefox and adobe reader, and compeltly balnk on atril

### ISSUE THREAD

**Issue by elyashiv** [mate-desktop/atril#195]
[לחם רב.PDF](https://github.com/mate-desktop/atril/files/208988/default.PDF)
This file looks fine in adobe reader and firefox, but completely blank on atril.

**Comment by raveit65** [comment]
works well here with atril on fedora
![bildschirmfoto zu 2016-04-07 20-50-25](https://cloud.githubusercontent.com/assets/961604/14363034/7bc19686-fd02-11e5-88d0-e632ec77ab42.png)

**Comment by elyashiv** [comment]
Doesn't work on mate 17 on a x86.
Running `atril --version` produces: `MATE Document Viewer 1.12.2`.
if you need more data, just ask.

![screenshot at 2016-04-07 21 54 55](https://cloud.githubusercontent.com/assets/1577529/14363175/8a810c0c-fd0b-11e5-82fb-f7dab81570c7.png)

**Comment by raveit65** [comment]
mate 17 ???

**Comment by elyashiv** [comment]
Oops, Linux mint 17. mistake.

**Comment by flexiondotorg** [comment]
Report this to Linux Mint, they package MATE independently and don't use the packages created by the Debian/Ubuntu packaging team.

**Comment by Fred-Barclay** [comment]
For what it's worth--it does fine on my system (LMDE Betsy MATE, atril version 1.12.2).
Moving over to the linuxmint/mate page...

**Comment by monsta** [comment]
Reproducible in Mint 17.3 with a bunch of messages in terminal:

```
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
Syntax Error: Missing 'endstream' or incorrect stream length
```
and so on.

**Comment by monsta** [comment]
Ok, google told me it's a problem in poppler (which is used as a PDF backend in Atril). There's no use in reporting bug in Mint repo. Report it on Launchpad against poppler package and ask the maintainers to apply the fix in Ubuntu 14.04 (as the issue is already fixed in 15.10 and newer).
